shared/src/commonTest/kotlin/co/touchlab/kampkit/BreedRepositoryTest.kt

```diff
@@ -5,31 +5,23 @@ import co.touchlab.kampkit.db.Breed
 import co.touchlab.kampkit.mock.ClockMock
 import co.touchlab.kampkit.mock.DogApiMock
 import co.touchlab.kampkit.models.BreedRepository
-import co.touchlab.kampkit.models.DataState
-import co.touchlab.kampkit.models.ItemDataSummary
 import co.touchlab.kermit.Logger
+import co.touchlab.kermit.StaticConfig
 import com.russhwolf.settings.MockSettings
 import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.FlowPreview
-import kotlinx.coroutines.flow.distinctUntilChanged
-import kotlinx.coroutines.flow.flattenMerge
-import kotlinx.coroutines.flow.flowOf
 import kotlinx.coroutines.runBlocking
 import kotlinx.coroutines.test.runTest
 import kotlinx.datetime.Clock
 import kotlin.test.AfterTest
 import kotlin.test.Test
 import kotlin.test.assertEquals
-import kotlin.test.assertNotNull
-import kotlin.test.assertTrue
+import kotlin.test.assertFails
 import kotlin.time.Duration.Companion.hours
-import kotlin.time.Duration.Companion.seconds
 
-@OptIn(FlowPreview::class)
 @RunWith(AndroidJUnit4::class)
 class BreedRepositoryTest {
```

Comment:
I think it's still hitting all the same cases (and some new ones), but I rewrote everything in here because a lot of the existing stuff was misnamed and confusing